frontend/src/scenes/actions/ActionsTable.js

```diff
@@ -108,91 +110,66 @@ export function ActionsTable() {
-                        events that you have decided to put into a manually-labelled bucket. They're used in Insights
-                        and Cohorts.
-                        <br />
-                        <br />
-                        <a href="https://posthog.com/docs/features/actions" target="_blank" rel="noopener noreferrer">
-                            See documentation
-                        </a>
-                    </div>
-                )}
-            </>
-        )
-    }
-
     return (
         <div>
-            <PageHeader title="Actions" caption={<Caption />} />
-            {featureFlags['actions-ux-201012'] && (
+            <div>
                 <div>
-                    <div className="tutorial-container">
-                        <div className="t-element">
-                            <div>
-                                <img src={imgGrouping} alt="" />
-                            </div>
-                            <div>
-                                <div className="title">Multiple grouping</div>
-                                <div className="description">Group multiple sets of events into a single action.</div>
-                            </div>
+                    Actions can retroactively group one or more raw events to help provide consistent analytics.{' '}
+                    <a href="https://posthog.com/docs/features/actions" target="_blank">
+                        <QuestionCircleOutlined />
+                    </a>
+                </div>
+                <div className="tutorial-container">
+                    <div className="t-element">
+                        <div>
+                            <img src={imgGrouping} alt="" />
                         </div>
-                        <div className="t-element">
-                            <div>
-                                <img src={imgStandardized} alt="" />
-                            </div>
-                            <div>
-                                <div className="title">Clean &amp; standardized data</div>
-                                <div className="description">
-                                    Keep your same actions even if your product or data changes.
-                                </div>
-                            </div>
+                        <div>
+                            <div className="title">Multiple grouping</div>
+                            <div className="description">Group multiple sets of events into a single action.</div>
                         </div>
-                        <div className="t-element">
-                            <div>
-                                <img src={imgRetroactive} alt="" />
+                    </div>
+                    <div className="t-element">
+                        <div>
+                            <img src={imgStandardized} alt="" />
```

Comment:
This is probably all over the place, so it's not about this PR necessarily, but we should have alts on images for accessibility and site scores and things

Reply:
Well I would argue that for this case the text underneath is the alt description, I don't think it makes sense to add extra information. Also, I don't think this will count towards scoring at all as this is behind authentication.